assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -35,18 +35,25 @@ export default function useExistingTagEffect() {
 	const existingTag = useSelect( ( select ) =>
 		select( MODULES_TAGMANAGER ).getExistingTag()
 	);
+	const containerID = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).getContainerID()
+	);
 	const existingTagPermission = useSelect( ( select ) =>
 		select( MODULES_TAGMANAGER ).getTagPermission( existingTag )
 	);
 	const hasExistingTagPermission = useSelect( ( select ) =>
 		select( MODULES_TAGMANAGER ).hasExistingTagPermission()
 	);
 	// Set the accountID and containerID if there is an existing tag.
-	const { selectAccount, selectContainerByID } = useDispatch(
+	const { selectAccount, selectContainerByID, setUseSnippet } = useDispatch(
 		MODULES_TAGMANAGER
 	);
 	useEffect( () => {
 		( async () => {
+			if ( hasExistingTag && existingTag === containerID ) {
+				// Disable the plugin snippet as we already show the tag via other means.
```

Comment:
Existing tags are by definition not from Site Kit so this comment is somewhat misleading as "we" are not the ones showing it 😄 
```suggestion
				// Disable the plugin snippet to avoid duplicate tagging.
```